atomic_scanners/pipeline-scanner/pipeline-scanner

```diff
@@ -3,8 +3,8 @@ scanner_name: pipeline-scanner
 image_name: registry.centos.org/pipeline-images/pipeline-scanner
 default_scan: redhat-release
 scans: [
-  { name: redhat-release,
+  { name: pipeline-scanner,
```

Comment:
Image name `pipeline-scanner` will not be affected by changing the name of a scan. In fact, I think, it won't be affected even if we changed the scanner name. Because we're specifying the image to be used.

Adding detail in the description sounds ok to me but if we think about it, scan named `pipeline-scanner` would only provide yum update facility. Doesn't sound appropriate to me.

Same image can be used to do more than one type of scan under the scanner `pipeline-scanner`. As a result, having appropriate name for a scan type makes more sense to me. Here's an example of single image doing multiple scans:

~~~
$ atomic scan --list
Scanner: openscap
  Image Name: rhel7/openscap
     Scan type: cve *
     Description: Performs a CVE scan based on known CVE data

     Scan type: standards_compliance
     Description: Performs a standard scan


* denotes defaults
~~~